crates/summit/src/routes.rs

```diff
@@ -67,6 +71,16 @@ pub async fn tasks(
     let end = (page + side_window).min(total_pages);
     let pages_to_show = (start..=end).collect::<Vec<u32>>();
 
+    if let (Some(sort), Some(order)) = (&selected_sort, &selected_order) {
+        let is_asc = order == "asc";
+
+        match sort.as_str() {
+            "ended" => sort_tasks_by_ended(&mut task_query, is_asc),
+            "build" => sort_tasks_by_build_time(&mut task_query, is_asc),
+            _ => {}
+        }
+    }
```

Comment:
Sorting needs to happen at the database level since we have pagination

Reply:
Well that is embarrassing, that is what I get for testing only on the first page with a few completed items and forgetting all about pagination.